### ISSUE TITLE
Small window title bar dragging issue

### ISSUE THREAD

**Issue by stefnotch** [FlaxEngine/FlaxEngine#63]
**Issue description:**
The dockable windows, like the material windows, are a bit odd when dragging. They sometimes jump to a different position when you drag them.

This happens because in the engine core, the mouse position doesn't get updated as soon as the mouse is above a window title bar. 

This piece of code doesn't get triggered. I'd gladly fix it, but am unsure how to best proceed.
https://github.com/FlaxEngine/FlaxEngine/blob/da389eea03ab0775e384d50ff65bf4d4792f3e47/Source/Engine/Platform/Windows/WindowsInput.cpp#L198-L203

**Steps to reproduce:**

1. Open a material window
2. Move your cursor below the title bar, at the very left of the window
3. Move your cursor onto the title bar 
4. Move your cursor all the way to the right on the title bar. Make sure to stay on top of the title bar while moving.
5. Start dragging
6. The window jumps

![image](https://user-images.githubusercontent.com/10220080/103142289-ffac8300-4700-11eb-9114-cf9526555a7e.png)
Basically just follow that mouse path ^ and start dragging

**Flax version:**
Flax 1.0

**Comment by stefnotch** [comment]
One way to fix it would be to add
```cpp
    case WM_NCMOUSEMOVE:
    {
        OnMouseMove(Vector2(static_cast<float>(static_cast<LONG>(WINDOWS_GET_X_LPARAM(lParam))), static_cast<float>(static_cast<LONG>(WINDOWS_GET_Y_LPARAM(lParam)))), window);
        result = true;
        break;
    }
```
